### ISSUE TITLE
Adjust Early Return Behavior for Fullscreen and Minimized States in BiDi Spec  (SetClientWindowState)

### ISSUE THREAD

**Issue by Temidayo32** [w3c/webdriver-bidi#839]
### Description:  
The current BiDi specification states the following in [`set the client window state`](https://w3c.github.io/webdriver-bidi/#set-the-client-window-state):  

1. Let current state be get the client window state with window.  
2. If current state is equal to state, return success with data null.  

This behavior prevents resizing or moving a window in the `Normal` state due to the early return.  

A suggestion will be to allow early return only for `Fullscreen` and `Minimized` states when the requested state matches the current state. This adjustment ensures flexibility for `Normal` state operations while optimizing behavior for `Fullscreen` and `Minimized` states.

**Comment by whimboo** [comment]
@Temidayo32 thank you for filing the issue. Would you be interested as well to provide a pull request for it? Therefore you will have to [join the W3C group first](https://www.w3.org/groups/wg/browser-tools-testing/join/).

**Comment by Temidayo32** [comment]
> @Temidayo32 thank you for filing the issue. Would you be interested as well to provide a pull request for it? Therefore you will have to [join the W3C group first](https://www.w3.org/groups/wg/browser-tools-testing/join/).

Yes, I will. I have created an account. Account awaiting further review/.

**Comment by whimboo** [comment]
@Temidayo32 you can go ahead and create a PR with the fix without a membership. Just fixing a bug should not actually require to be a member.

**Comment by whimboo** [comment]
Fixed by https://github.com/w3c/webdriver-bidi/pull/844. Thanks @Temidayo32!
